src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -76,12 +76,19 @@ export default View.extend({
     `,
   getTemplateData() {
     const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    let enrolledDeviceName = '';
+    if (contextualData && contextualData?.devicebootstrap && contextualData?.devicebootstrap.enrolledDevices) {
```

Comment:
Not familiar with js, but when would `contextualData?.devicebootstrap.enrolledDevices` return true? And what is the type? Would it return true in case of empty string?
Asking because below code would return enrolledDevices in the raw form if it cannot be converted to an array.

Reply:
May i know why we specifically parse the devices name here instead of just using the old way? any issue you spot with the old way?